relay-aws-extension/mock/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Create virtual Python environment
+python -m venv .venv
+source .venv/bin/activate
+
+# Install Python requirements
+pip install -r requirements.txt
+
+# Start mock API server
+FLASK_APP="mock-aws-lambda-extensions-api" flask run
```

Comment:
Since we're already running flask apps in integration tests (see `tests/integration/*`), what if you move this flask app as fixture there, and then create a new integration test file for this?